app/services/exports/export_distributions_csv_service.rb

```diff
@@ -118,7 +118,7 @@ def base_headers
     def item_headers
       return @item_headers if @item_headers
 
-      @item_headers = @organization.items.order(:created_at).distinct.select([:created_at, :name]).map(&:name)
+      @item_headers = @organization.items.select("DISTINCT ON (LOWER(name)) items.name").order("LOWER(name) ASC").map(&:name)
```

Comment:
I'm pretty sure there's a more rails-y way to do this.      I haven't tried it out but it might be: 
 @organization.items.order('lower(name)').distinct.select([:created_at, :name]).map(&:name)

Reply:
Ok - so maybe I was wrong *g* .  @awwaiid  -- do you have a quick take on this?

Reply:
I asked a robot (Claude) and it suggested
```
@item_headers = @organization.items
  .select(:name)
  .group('LOWER(name)')
  .order('LOWER(name) ASC')
  .pluck('MIN(name)')
```
which isn't a direct translation but is actually sounds pretty good (it doesn't instantiate Item). mmm. OK, I asked it for something more direct and it gave
```
@item_headers = @organization.items
  .select('DISTINCT ON (LOWER(items.name)) items.name')
  .order('LOWER(items.name) ASC')
  .pluck(:name)
```
which is back to basically what you have now.

I think since there isn't really a direct ActiveRecord thing for `distinct on` like this that what you have is fine, though you should consider the `.pluck(:name)` instead of the `.map(&:name)`

Reply:
![image](https://github.com/user-attachments/assets/4614562a-69c2-4073-932a-6d3e1cfd65b4)

After running a benchmark (screenshot), it turns out that map is actually a bit faster than pluck, though the difference isn't huge. Is it still better to stick with pluck, or should we consider using map? Both perform well, so it might come down to personal preference or readability. Let me know what you think!